artiq/frontend/artiq_rtiomon.py

```diff
@@ -6,13 +6,38 @@
 from artiq.coredevice.comm_moninj import *
 
 
+class NumberRangeParser(argparse.Action):
```

Comment:
I don't think this complicated and potentially buggy parser is very useful. Specifying channels one by one would be just fine I think.